tests/main/interfaces-wayland/task.yaml

```diff
@@ -0,0 +1,52 @@
+summary: Ensure that the wayland interface works
+
+# Only test on classic Ubuntu amd64 systems that have wayland
+systems: [ ubuntu-1*-*64 ]
+
+prepare: |
+    . $TESTSLIB/pkgdb.sh
+    echo "Ensure we have a wayland compositor"
+    distro_install_package weston
```

Comment:
This should go in the pkgdb.sh script, we started to install all the dependencies in the project preparation a time ago.